src/toolkit/mat_query.h

```diff
@@ -35,6 +35,9 @@ class MatQuery {
   /// Returns the mass fraction of nuclide nuc in the material.
   double mass_frac(std::string nuc);
 
+  /// Returns the mass fraction of nuclide nuc in the material.
+  double multi_mass_frac(std::set<Nuc> nucs);
```

Comment:
I think I would prefer this to simply overload `mass_frac`, as the "multi" part is implied in the function argument. Do others have an opinion?